packages/iov-bns/src/decode.ts

```diff
@@ -107,6 +107,13 @@ function decodeVersionedId(versionedId: codecImpl.orm.IVersionedIDRef): Versione
   };
 }
 
+function decodeVersionedIdArray(versionedId: Uint8Array): VersionedId {
+  return {
+    id: Uint64.fromBytesBigEndian(versionedId.slice(0, 8)).toNumber(),
+    version: Uint64.fromBytesBigEndian(versionedId.slice(8, 16)).toNumber(),
```

Comment:
The final imlementation ended up encoding the versin as Uint32 (4 bytes): https://github.com/iov-one/weave/pull/950/files, so we can use `Uint32` instead

Reply:
To clarify: 12 bytes total (8 for id, 4 for version)?

Reply:
Exactly